suite/tests/CMakeLists.txt

```diff
@@ -2305,7 +2305,14 @@ if (CLIENT_INTERFACE)
                 "-thread_private -prof_pcs -opt_cleancall 0" "")
             endif (NOT X64)
           endif (UNIX AND NOT ARM)
-        endif ()
+        elseif (sample STREQUAL "memval_simple")
+          # i#2449: only witnessed crash on 32-bit linux
```

Comment:
True, x86_64 should just work. However, I assumed that enabling this test for 64-bit would not yield much information, as we could never get `memval_simple` to crash on 64-bit to begin with (probably due to a drastic increase in the number of registers available to `drreg`). If you'd like I can enable it in a new PR.